test/com/moclojer/adapters_test.clj

```diff
@@ -9,33 +9,39 @@
       {:config-path "config.yaml"
        :mocks-path "mocks.yaml"
        :version true
-       :help true}
+       :help true
+       :format "println"}
       (adapters/inputs->config {:args [{:config "config.yaml"
                                         :mocks "mocks.yaml"
                                         :version true
                                         :help true}]
                                 :opts {:config "opts-config.yaml"
-                                       :mocks "opts-mocks.yaml"}}
+                                       :mocks "opts-mocks.yaml"
+                                       :format "println"}}
                                {:config "default-config.yaml"
                                 :mocks "default-mocks.yaml"})
 
       {:config-path "opts-config.yaml"
        :mocks-path "opts-mocks.yaml"
        :version true
-       :help true}
+       :help true
+       :format "json"}
       (adapters/inputs->config {:args []
                                 :opts {:config "opts-config.yaml"
                                        :mocks "opts-mocks.yaml"
                                        :version true
-                                       :help true}}
+                                       :help true
+                                       :format "json"}}
```

Comment:
The tests are not actually validating if the stdout output is in JSON. Can you think of a way to do this test?

Reply:
Sure! I'll be covering this usecase asap